src/Transformers/NumericStringConverter.php

```diff
@@ -14,21 +14,12 @@
  * Useful for when extracting from a source that only recognizes data as string
  * types such as CSV.
  *
- * **Note:** The string representation of the PHP constant NAN (not a number) is `NaN`.
- *
  * @category    Machine Learning
  * @package     Rubix/ML
  * @author      Andrew DalPino
  */
-class NumericStringConverter implements Transformer
+class NumericStringConverter implements Transformer, Reversible
 {
-    /**
-     * The numeric string representation of NaN.
-     *
-     * @var string
-     */
-    public const NAN_PLACEHOLDER = 'NAN';
```

Comment:
Should the removal of this constant be mentioned in the CHANGELOG as a BC break?

Reply:
Yes, good call ... it's a public member of the class so definitely